packages/react/src/components/info-button/readme.md

```diff
@@ -0,0 +1,38 @@
+# `<DsoInfoButton>`
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Properties
```

Comment:
Ik denk dat je hier te enthousiast hebt gekopieerd, want in de react package genereert Stencil geen readme.md